subcommand/server-acl-init/create_or_update.go

```diff
@@ -118,9 +118,18 @@ func (c *Command) checkAndCreateNamespace(ns string, consulClient *api.Client) e
 
 	// If not, create it
 	if namespaceInfo == nil {
+		// Create the ACLs config for the cross-Consul-namespace
+		// default policy that needs to be attached
+		aclConfig := api.NamespaceACLConfig{
+			PolicyDefaults: []api.ACLLink{
+				{Name: "cross-namespace-policy-for-k8s"},
```

Comment:
A slight nit: do we need `-for-k8s` in the name? There's no way we could limit this policy to only-k8s services (nor should we), and I think the description is enough to show that the helm chart created it.

Reply:
Switched to `cross-namespace-policy`.